### ISSUE TITLE
IN PROGRESS: 0852IbnHajarCasqalani.FathBari.Shamela0001673-ara1

### ISSUE THREAD

**Issue by Gowaart** [OpenITI/Annotation#1039]
URI: 0852IbnHajarCasqalani.FathBari.Shamela0001673-ara1
ANNOTATOR: Gowaart

**Comment by hamidrh66** [comment]
I just had a quick glance at this text for the geography project, without intending to tag it. It’s a very complicated text. What I understood is that all the parts that are quotations from Bukhari are intentionally omitted in the digital version. Besides, the text seems to be in a bad condition. The page numbers are not correct. Page 8 which is the beginning of the text appear after PageV01P492 and I couldn’t understand what is going on in some parts. It needs a very careful collation with the PDF: https://archive.org/details/FathAlBariTaliqBinBaz/000

**Comment by pverkind** [comment]
The Shamela file is very problematic; unfortunately, the Sham19Y version of the file is not better. 
As for JK: this one has the pagination of https://archive.org/details/FathAlBariTaliqBinBaz/000, but has very little signposting of the structure (in addition, as HRH pointed out, it leaves out Bukhārī's text at the beginning of each chapter, which is in fact OK, I think, as we try to have only one text per file, and the text is entirely understandable without these quotations). 
The Shia text is another edition, with different page numbers.

**Comment by Gowaart** [comment]
I don't think the leaving out of Bukhārī's text is unproblematic though. Considering it's a sharh, isn't it crucial to have the original text available as well? Furthermore, other sharhs of Bukhari have retained Bukhari's text in their digital versions so the reuse statistics would be inconsistent.

**Comment by pverkind** [comment]
I agree consistency across the corpus is a problem. But I do think that having the texts separated for all sharḥs, ḥāshiyas etc. is the way to go. If we separate the commented-upon text from the commentary, it is still possible to read them side by side since they have the same pagination (in the case of this specific text, we cannot do that because the main text is not in the file).

**Comment by pverkind** [comment]
@maximromanov @Lrnzmtths

**Comment by pverkind** [comment]
@SarahKITAB @aslishah @mabarber92

**Comment by aslishah** [comment]
If there was a way, it would be best to keep the digital edition as close to the actual manuscript as possible.
